### PR TITLE
Ku/merge 2.0

### PR DESCRIPTION
**Pull request by kuenishi**
To include #1255 and #1272 in 2.1.

### CODE REVIEW

File: src/riak_cs_xml.erl [hunk 1]
```diff
@@ -143,26 +150,34 @@ owner_content({OwnerName, OwnerId}) ->
...
+                {'Marker',     [Resp?LORESP.marker]}] ++
                 %% use a list-comprehension trick to only include
                 %% the `NextMarker' element if it's not `undefined'
-               [make_external_node('NextMarker', NextMarker) ||
-                NextMarker <- [Resp?LORESP.next_marker],
-                NextMarker =/= undefined,
-                Resp?LORESP.is_truncated] ++
-               [make_external_node('MaxKeys', Resp?LORESP.max_keys),
-                make_external_node('Delimiter', Resp?LORESP.delimiter),
-                make_external_node('IsTruncated', Resp?LORESP.is_truncated)] ++
```
Comment (shino) [inline src/riak_cs_xml.erl:162]: Sorry for stepping in... this line may be necessary, it was added for bug fix at https://github.com/basho/riak_cs/commit/45387fd2afb9be80379caf25eacb7cc4eebf026b?w=1

Comment (shino) [inline src/riak_cs_xml.erl:162]: Ugh, commented on wrong line...

File: src/riak_cs_xml.erl [hunk 2]
```diff
@@ -143,26 +150,34 @@ owner_content({OwnerName, OwnerId}) ->
...
+                         CommonPrefix <- Resp?LORESP.common_prefixes],
+    Contents = [{'Name',       [Resp?LORESP.name]},
+                {'Prefix',     [Resp?LORESP.prefix]},
+                {'Marker',     [Resp?LORESP.marker]}] ++
                 %% use a list-comprehension trick to only include
                 %% the `NextMarker' element if it's not `undefined'
-               [make_external_node('NextMarker', NextMarker) ||
-                NextMarker <- [Resp?LORESP.next_marker],
-                NextMarker =/= undefined,
-                Resp?LORESP.is_truncated] ++
```
Comment (shino) [inline src/riak_cs_xml.erl:159]: I meant this condition in the comment :arrow_down:

**Comment by kuenishi** [comment]
It might be strange that I do +1 to my pull request, but according to my test on inspector it has worked fine.

**Comment by kuenishi** [comment]
@borshop merge

**Pull request merged**